content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -216,17 +269,17 @@ If this error message appears after rebooting, install the Linux Kernel Update P
 
 ![DockerDesktopError](/images/TrueCommand/DockerDesktopError.png "Docker Desktop Error")
 
-The update package is downloaded from a [Windows storage location](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi).
+The update package downloads from a [Windows storage location](https://wslstorestorage.blob.core.windows.net/wslblob/wsl_update_x64.msi).
 
 ![DockerDesktopInstallWSL](/images/TrueCommand/DockerDesktopInstallWSL.png "Docker Desktop Install WSL")
 
-Microsoft has additional documentation available for assistance with [downloading the Linux kernel update](https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-4---download-the-linux-kernel-update-package).
+Microsoft has additional documentation available to assist with [downloading the Linux kernel update](https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-4---download-the-linux-kernel-update-package).
 
 After installing the kernel update, set the WSL default version to `version 2` by opening the Powershell and entering `wsl --set-default-version 2`.
```

Comment:
maybe this instead, because we don't need to say the `version 2` in CLI emphasis because it is not a variable we are telling them exactly what command string to type so just remove it.
```suggestion
After installing the kernel update, set the WSL default version by opening the Powershell and entering `wsl --set-default-version 2`.
```